source/manual/common-aws-tasks-for-2nd-line-support.html.md

```diff
@@ -207,6 +187,33 @@ old persistent disk
 1. Reboot the machine to make sure the persistent disk reattaches between reboots
 1. Initiate a puppet run to make sure that everything works
 
+## Users
+
+### Add a developer as an AWS admin
+
+_**Note**: This task doesn't need to be done by 2nd line._
+
+_**Note**: Users with production access can already assume an admin role
+in every environment. Read [assuming all roles for users with production access][]._
+
+You can modify the AWS roles that a developer can assume in the
+[govuk-aws-data][] repository.
+
+Typically this involves adding a user ARN (a line containing the person's email
+address) to a list named `role_internal_admin_user_arns`.
+
+See these two example PRs:
+
+* https://github.com/alphagov/govuk-aws-data/pull/837
+* https://github.com/alphagov/govuk-aws-data/pull/819
+
+Once you have put up a PR to change the developer's role, you'll need to assume
+the admin role in order to [apply the Terraform change][].
+
+[assuming all roles for users with production access]: https://docs.publishing.service.gov.uk/manual/get-started.html#assuming-all-roles-for-users-with-production-access
+[govuk-aws-data]: https://github.com/alphagov/govuk-aws-data
+[apply the Terraform change]: https://docs.publishing.service.gov.uk/manual/deploying-terraform.html
+
```

Comment:
I think we can probably remove this section of docs, since (as per the note), everyone with production access has permission to assume the admin role now.

Reply:
I did wonder about the usefulness of this section - removing it sounds good to me 👍